docs/gettingStarted.md

```diff
@@ -10,8 +10,7 @@ To use HubTurbo, you'll need:
 
 ## Installation
 
-Once you have the above, grab the installer from [here](https://github.com/HubTurbo/AutoUpdater/releases/download/V0.9.0/HubTurbo.jar) and run it. 
-It will download the latest HubTurbo version and launch it. 
+Once you have the above, grab HubTurbo from [here](https://github.com/HubTurbo/HubTurbo/releases/download/V3.22.0/resource-v3.22.0.jar) and run it.
```

Comment:
Does this need to be updated every time? Maybe we could link to [this](https://github.com/HubTurbo/HubTurbo/releases/latest) page instead.